pkg/util/util.go

```diff
@@ -50,3 +57,31 @@ func EqualStringSlice(a, b []string) bool {
 	}
 	return true
 }
+
+func GetCmdLine(pid int) (string, error) {
```

Comment:
this is only used by `harvest.go` maybe premature to move to a util and make public

Reply:
this looks like a util. I like this to be in util.